CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 2022-03
+Support for Babelfish v.1.2.0
+
+New -recursive option to process directory trees
+
+New -include and -exclude options to filter on file type
+
+Fix for processing UTF8-with-BOM input files
+
+Various improvements to grammar and reporting
```

Comment:
```suggestion
- New -recursive option to process directory trees

- New -include and -exclude options to filter on file type

- Fix for processing UTF8-with-BOM input files

- Various improvements to grammar and reporting
```

Reply:
Thanks. Updated the CHANGELOG.md with your suggestions.